src/app/(auth)/signup/page.tsx

```diff
@@ -99,40 +107,48 @@ export default function SignUp() {
               password={password}
               setComplexity={setPasswordComplexity}
             />
-          </SpacerDiv>
-          <SpacerDiv>
-            <BigBlueButton type="button" onClick={handleSignUp}>
-              Sign Up
-            </BigBlueButton>
+          </Flex>
+          <TextInput
+            label="Confirm Password"
+            placeholder="Confirm Password"
+            type="password"
+            id="confirmPassword"
+            value={confirmPassword}
+            setValue={setConfirmPassword}
+          />
+          <Flex $direction="column" $gap="20px">
+            <BigBlueButton type="submit">Sign Up</BigBlueButton>
             <H4Centered>
               Have an account already?{' '}
               <LinkColored $color={COLORS.greyDark} href="/login">
                 Log In
               </LinkColored>
             </H4Centered>
-          </SpacerDiv>
-        </>
+          </Flex>
+        </SmallCardForm>
       )}
       {emailSentCount > 0 && (
-        <SpacerDiv>
-          <H2>An email verification link has been sent.</H2>
-          <H4 $color={COLORS.greyDark}>
-            This link will direct you to the next step. If you didn’t receive an
-            email, please click Resend Email.
-          </H4>
-          <HorizontalDiv>
-            <Button
-              $primaryColor={COLORS.blueMid}
-              $secondaryColor={COLORS.blueDark}
-              onClick={handleResendEmail}
-            >
-              <H4 $color="white">Resend Email</H4>
-            </Button>
-            {emailSentCount > 1 && (
-              <P $color={COLORS.greyMid}>Email has been resent!</P>
-            )}
-          </HorizontalDiv>
-        </SpacerDiv>
+        <SmallCardForm onSubmit={handleSignUp}>
+          <Flex $direction="column" $gap="20px">
+            <H2>An email verification link has been sent.</H2>
+            <H4 $color={COLORS.greyDark}>
+              This link will direct you to the next step. If you didn’t receive
+              an email, please click Resend Email.
+            </H4>
+            <HorizontalDiv>
+              <Button
+                $primaryColor={COLORS.blueMid}
+                $secondaryColor={COLORS.blueDark}
+                onClick={handleResendEmail}
+              >
```

Comment:
similar: i think there's a `BlueButton` we can replace this with, or otherwise we should set `$tertiaryColor` to `COLORS.blueDarker`

Reply:
let's use `BigBlueButton`!